pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueImageEdit/VenueImageEdit.tsx

```diff
@@ -15,22 +15,26 @@ const CROP_BORDER_HEIGHT = 40
 const CROP_BORDER_WIDTH = 100
 const CROP_BORDER_COLOR = '#fff'
 
-type Props = {
+type VenueImageEditProps = {
```

Comment:
Ca n'est pas le sujet de ta PR autant nettoyer au passage.
```suggestion
interface IVenueImageEditProps {
```